app/(afterLogin)/(planningAndReporting)/planning-and-reporting/_components/planning/createPlanHierarchy.tsx

```
@@ -108,200 +114,203 @@ const PlanningHierarchyComponent: React.FC<CollapseComponentProps> = ({
+                              >
+                                Add Plan Task
+                              </Button>
+                              {task?.achieveMK && (
+                                <Tooltip title="Plan Milestone as a Task">
+                                  <Button
+                                    id="plan-milestone-as-task"
+                                    disabled={
+                                      statuses[milestone?.id] ||
+                                      milestone?.status === 'Completed' ||
+                                      Number(keyResult?.progress) === 100 ||
+                                      ((form?.getFieldValue(`names-${compositeKey}`)?.length ?? 0) > 0) ||
```

ESLint

[error] Replace `(form?.getFieldValue(`names-${compositeKey}`)?.length·??·0)·>·0)` with `form?.getFieldValue(⏎········································`names-${compositeKey}`,⏎······································)?.length·??·0)·>·0`

(prettier/prettier)